kong/router/atc_compat.lua

```diff
@@ -107,10 +120,10 @@ local OP_REGEX = "~"
 local function get_atc(route)
```

Comment:
if all router's params (hosts/headers/...) are nil, then thsis function will return an empty string.

Could we deal with this situation in act-router?